release/long_running_tests/workloads/serve_failure.py

```diff
@@ -30,6 +30,11 @@
 
 RAY_UNIT_TEST = "RAY_UNIT_TEST" in os.environ
 
+# RAY_OVERRIDE_RESOURCES will prevent "num_cpus" working
+# when adding customized resource node to Cluster.
+if "RAY_OVERRIDE_RESOURCES" in os.environ:
+    del os.environ["RAY_OVERRIDE_RESOURCES"]
+
```

Comment:
hmm this seems like a hack -- should we remove where it's being set from instead?